doc/classes/@GDScript.xml

```diff
@@ -250,7 +250,7 @@
 			<argument index="0" name="step" type="float">
 			</argument>
 			<description>
-				Returns the number of digit places after the decimal that the first non-zero digit occurs.
+				Returns the offset (after the decimal point) of the first non-zero digit. The returned offset starts at 1.
```

Comment:
Is offset the right word to use here? I'm not sure what I would use this method for but would probably involve the total digits not the place of the digits which to me makes the use of offset seem wrong.

Reply:
I agree, it's more something of the scale of the smallness of the fractional part. Although that doesn't sound really nice either. I'll try to make a better sentence of it without 'offset'.

Reply:
Perhaps :
Returns the position of the first non-zero digit, after the decimal point.
?